addons/languages/english/pack/src/main/res/xml/terminal_with_symbols.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I feel like we can just make the changes in the original `terminal` layout. Why do we need two terminal layouts?

Reply:
I agree with you. I would do the same with the italian one as well.

Reply:
thanks

Reply:
Done for terminal. Should I do the same for italian?